sway/commands/output.c

```diff
@@ -26,6 +27,13 @@ struct cmd_results *cmd_output(int argc, char **argv) {
 		return error;
 	}
 
+	// The NOOP-1 output is a dummy output used when there's no outputs
+	// connected. It should never be configured.
+	if (strcasecmp(argv[0], "NOOP-1") == 0) {
```

Comment:
Can we check against `root.noop_output->name` instead? I'm worried about someone using the noop beackend of unit tests and running into this.